backend/handler.js

```diff
@@ -54,7 +55,7 @@ module.exports.viewPins = async (event) => {
 };
 
 module.exports.viewCategories = async (event) => {
-  if (event.path === '/dev/yourPins/categories' && event.httpMethod === 'GET') {
+  if (event.path === '/yourPins/categories' && event.httpMethod === 'GET') {
```

Comment:
When using serverless, by default, it puts all of the endpoints or routes that you define in serverless.yml with the `/dev` prefix, but your handler won't know about it.

That means when your handler receives an event, it will see the event.path as only `/yourPins/categories` and not `/dev/yourPins/categories`.